guides/Getting-Started.md

```diff
@@ -16,20 +16,21 @@ The Purescript compiler (`purs`) can be installed with npm:
 
 #### Setting up the Development Environment
 
-PureScript's core libraries are configured to use the [Pulp](https://github.com/purescript-contrib/pulp) build tool, and packages are available in the [Bower registry](http://bower.io/search/?q=purescript-).
+PureScript's core libraries are configured to use the [Pulp](https://github.com/purescript-contrib/pulp) build tool, and packages are installed with [psc-package](https://psc-package.readthedocs.io/). You can find the packages curated packages on [Pursuit](https://pursuit.purescript.org/)
```

Comment:
>  You can find the packages curated packages on [Pursuit (https://pursuit.purescript.org/)

This seems confusing, that it would make people think that all Pursuit packages are in package sets. Also that package sets aren't actually curated, just community contributed.